README.md

```diff
@@ -2,7 +2,7 @@
 
 # Development
 ## Requirements
-See [the main project requirements](https://github.com/nuxeo/doc.nuxeo.com/blob/master/README.md#requirements).
```

Comment:
I'm happy to have both references if you think that's clearer.

The reason the requirements are different is because doc.nuxeo.com needs libsass and the content repos technically don't.